beautifultable/beautifultable.py

```diff
@@ -1213,3 +1216,47 @@ def from_csv(self, file_name, header=True, **kwargs):
             for row in csv_reader:
                 self.rows.append(row)
             return self
+
+    def to_df(self):
+        """Export table to dataframe.
+
+        Parameters
+        ----------
+        table : Beautifuletable
+
+        Raises
+        ------
+        AttributeError
+            If header is not there in table
+        """
+
+        headers = self.columns.header
+        headers = list(headers)
+        if not headers:
+            raise AttributeError("Dataframe needs a header")
```

Comment:
do you think a ValueError might be more appropriate here?

Reply:
I realized just now that pandas pick default values for columns and indexes.

Reply:
I see that you had to check if all members are null to identify the case of no headers. there should be a simpler way for that. I'll keep this in mind.